Kinvey/Kinvey/DataStore.swift

```diff
@@ -299,9 +326,38 @@ open class DataStore<T: Persistable> where T: NSObject {
         return request
     }
     
+    /**
+     Custom aggregation function performed in the backend.
```

Comment:
What do you mean by `performed in the backend`? If this API only works against the backend, you should say that specifically instead.

"Custom aggregation function. Note: this function does not work on local data. It must be run against the backend."